sopc_components_altera.xml

```diff
@@ -1,4 +1,16 @@
 <SOPC2DTS_COMPONENTLIB vendor="ALTR">
+	<S2DComponent classname="mm_ccb"
+		group="bridge">
+	</S2DComponent>
+	<S2DComponent classname="altera_i2cslave_to_avlmm_bridge"
+		group="bridge">
+	</S2DComponent>
+	<S2DComponent classname="altera_up_avalon_to_external_bus_bridge"
+		group="bridge">
+	</S2DComponent>
+	<S2DComponent classname="altera_vic"
+		group="ignore">
+	</S2DComponent>
 	<S2DComponent classname="altera_nios2,altera_nios2_qsys" group="cpu" compatDevice="nios2">
```

Comment:
You can locally ignore this one (custom component lib) but I think we should probably keep this one un-ignored to help "motivate" someone to write a driver for this.

Reply:
When you refer to a driver, do you mean a Linux driver? Or some spoc2dts extension?  
How would that look like on the XML if we had a driver?

Reply:
I mean a Linux driver (haven't checked, there might be one).
What it would look like is depending on what info that driver would need.

Reply:
> What it would look like is depending on what info that driver would need.

Ok - I've commented it out, so it would stand out for anyone who might be writing a driver.  

My use case is actually different. I'm not using the device tree for Linux kernel. I'm reading the dtb file in QEMU to detect the devices and their parameters, then instantiate and connect them to a nios2 cpu simulation.

Reply:
Ah, that's cool!
Didn't know that was possible (haven't used Nios2 in quite a few years tbh). Is there some public info about that somewhere?

Reply:
> Is there some public info about that somewhere?

I don't think so... Basically during the QEMU board initialization phase I'm using libfdt (which is already part of QEMU) to iterate the dtb file and configure/connect nios2 devices accordingly.   
This is useful with nios2 when FPGA bit-file changes frequently and we want the QEMU simulation to always be aligned with the latest sopcinfo.